### PR TITLE
Adding dbt operator

### PR DESCRIPTION
**Pull request by siklosid**
Adding dbt operator

### CODE REVIEW

File: dagger/dag_creator/airflow/operator_creators/dbt_creator.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            [f"{param_name}:{param_value}" for param_name, param_value in self._template_parameters.items()]
+        )
+
+        if len(self._template_parameters) > 0:
+            command += [f"--vars {dbt_vars}"]
+
+        return " ".join(command)
+
+    def _create_operator(self, **kwargs):
+        full_bash_command = f"{self._generate_build_command()}; {self._generate_build_command()}"
```
Comment (jorgetagle) [inline dagger/dag_creator/airflow/operator_creators/dbt_creator.py:46]:
```suggestion
        full_bash_command = f"{self._generate_deps_command()}; {self._generate_build_command()}"
```

File: dagger/pipeline/tasks/dbt_task.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                Attribute(
+                    attribute_name="profile_dir",
+                    parent_fields=["task_parameters"],
+                    comment="Path to the dbt profile dir",
+                ),
+                Attribute(
+                    attribute_name="profile_name",
+                    required=False,
+                    parent_fields=["task_parameters"],
+                    comment="Dbt profile name passed into dbt --target, default is default'",
```
Comment (jorgetagle) [inline dagger/pipeline/tasks/dbt_task.py:26]:
```suggestion
                    comment="Dbt profile name passed into dbt --target, default is 'default'",
```

File: dagger/dag_creator/airflow/operator_creators/dbt_creator.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+            "dbt build",
+            f"--project-dir {self._project_dir}",
+            f"--profiles-dir {self._profile_dir}",
+            f"--target {self._profile_name}",
+        ]
+
+        if self._select:
+            command += [f"--select {self._select}"]
+
+        dbt_vars = ",".join(
```
Comment (jorgetagle) [inline dagger/dag_creator/airflow/operator_creators/dbt_creator.py:36]: As a minor improvement, you can move this instruction inside the if statement.

File: dagger/dag_creator/airflow/operator_creators/dbt_creator.py [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+            [f"{param_name}:{param_value}" for param_name, param_value in self._template_parameters.items()]
```
Comment (jorgetagle) [inline dagger/dag_creator/airflow/operator_creators/dbt_creator.py:37]: dbt vars are exactly built as required:
https://docs.getdbt.com/docs/building-a-dbt-project/building-models/using-variables#defining-variables-on-the-command-line

We need to add quotes and curly brackets, and handle any eventual quotes that are used in the arguments params.

Comment (siklosid) [reply dagger/dag_creator/airflow/operator_creators/dbt_creator.py:37]: Thanks, I found this doc: https://docs.getdbt.com/docs/build/project-variables#defining-variables-on-the-command-line, changed the code to generate the second version. Without quotes. If escape needed, we can add the escape to the parameter itself. But I would be surprised.

**Pull request merged**